airflow/executors/celery_executor.py

```diff
@@ -84,7 +84,7 @@ def execute_async(self, key, command,
         self.log.info("[celery] queuing {key} through celery, "
                       "queue={queue}".format(**locals()))
         self.tasks[key] = execute_command.apply_async(
-            args=[command], queue=queue)
+            args=command, queue=queue)
```

Comment:
@bolkedebruin 
Just found an issue for this commit. The update for function "apply_async" will crash scheduler when using celery executor. Tested in local environment.
Concern for update in line 87:
**Before**: 
>`args=[command]`  and command is a unicode type string . 
> samples: `command = "airflow run dag323..."` and at this time `args = ["airflow run example"]`
>`execute_command("airflow run dag323")` asynchronously
**After**: 
>`args=command` and command is a list of short unicode strings.
> samples: `command = ["airflow", "run", "dag323",...]` and now`args = ["airflow", "run", "dag323", ...]`
>`execute_command("airflow", "run", "dag323",...)` ** Error here**

`execute_command.apply_async(args = command, queue = queue)` will pass a list of arguments rather than one to function `execute_command` which is defined as taking only one argument. The test in test_celery_executor.py right now only have one item in the testing `command` list and can not cover this case.

Reply:
@YingboWang , good find. Do you want to create a pr for this issue?

Reply:
I would like to.

Reply:
Good catch! I do think renaming the parmeter is the right way